learn/testing-ballerina-code/executing-tests.md

```diff
@@ -0,0 +1,81 @@
+---
+layout: ballerina-left-nav-pages
+title: Executing Tests
+description: Learn how to use different options for executing Ballerina tests.
+keywords: ballerina, programming language, testing
+permalink: /learn/testing-ballerina-code/executing-tests/
+active: executing-tests
+redirect_from:
+  - /learn/testing-ballerina-code/executing-tests
```

Comment:
@Thimuth should we also include ` /v1-2/learn/testing-ballerina-code/executing-tests`?

Reply:
Added /v1-2/ as well to the redirect_from portion 

Refer commit `Update Testing Ballerina code links and images`